src/clojurebot/core.clj

```diff
@@ -41,6 +41,11 @@
   [name handler]
   (swap! commands assoc name handler))
 
+(defn replace-nl
+  "Replace occurences of \n from s with whitespace, so it doesn't trim the IRC output."
+  [s]
+  (clojure.string/replace s "\n " " "))
```

Comment:
I'd use `(clojure.string/replace s #"\s+" " ")`, this way you're using a regex to replace ALL whitespace.